src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -33,7 +33,8 @@ program_node::program_node(std::shared_ptr<primitive> prim, program& prog)
         num_outputs = prim->num_outputs;
         for (int i = 0 ; i < num_outputs; ++i) {
             layout output_layout = layout(data_types::f32, format::bfyx, tensor());
-            output_layout.data_padding = prim->output_paddings[i];
+            // TODO(kelvin): Apply each output padding by output index. ex) output_layout.data_padding = prim->output_paddings[i];
+            output_layout.data_padding = prim->output_paddings[0];
```

Comment:
why is this changed?"

Reply:
Multiple paddings for multiple output should be implemented but right now 0 index for all output_paddings is enough. I'd like to update this part at later.